lib/content_item_publisher/content_item_presenter.rb

```diff
@@ -27,7 +27,7 @@ def present
     end
 
     def present_links
-      { content_id: content_id, links: {} }
+      { content_id:, links: {} }
```

Comment:
I'm confused about how this is working... does it just automatically use a value with the same name as the key if its not specified?

Reply:
Yeah, exactly right! I find it kinda confusing to be honest. But, it does clear up the params nicely though.

Reply:
Whoever thought we'd be borrowing syntactic sugar from Javascript!

Also thanks for sorting this out - I dreaded having to do it myself as part of the search improvement work ❤️